messaging.md

```diff
@@ -308,4 +308,35 @@ To access participants that are only related to messaging, you can get them by
 val messagingParticipants = conversation.participants(AxpChannel.MESSAGING)
 ```
 
+### Monitoring Typing Participants
+
+You can monitor participants who are typing in a conversation by adding a 
+listener to the conversation or by observing the flow.
+
+```kotlin
+// Using a listener
+conversation.addTypingStartedListener { participant ->
+    // You can now use the participant who started typing
+}
+conversation.addTypingStoppedListener { participant ->
+    // You can now use the participant who stopped typing
+}
+```
+or
+
+```kotlin
+// Using flow
+conversation.typingParticipantsFlow.collect { typingParticipants ->
+    // You can now use the set of participants who are typing
+}
+```
+
+### Send typing status
+
+You can send typing status to the conversation by calling the `notifyUserTyping()` method whenever user is typing.
```

Comment:
We should mention the rate limiting here, so that the app authors know that it's safe to call this for every keystroke.